examples/lib/commons/circle_component.dart

```diff
@@ -1,28 +1,15 @@
-import 'dart:ui';
-
 import 'package:flame/components.dart';
 
-class CircleComponent extends PositionComponent {
-  CircleComponent({this.radius = 10.0})
-      : paint = Paint()..color = const Color(0xFF60CB35),
-        super(
-          size: Vector2.all(2 * radius),
-          anchor: Anchor.center,
-        );
-
-  Paint paint;
-  double radius;
-
-  @override
-  void render(Canvas canvas) {
-    super.render(canvas);
-    canvas.drawCircle(Offset(radius, radius), radius, paint);
-  }
-
-  @override
-  bool containsPoint(Vector2 point) {
-    final local = absoluteToLocal(point);
-    final center = Vector2.all(radius);
-    return local.distanceToSquared(center) <= radius * radius;
+class ExampleCircleComponent extends CircleComponent {
+  ExampleCircleComponent({
+    double radius = 10,
+    Vector2? position,
+    int priority = 0,
+  }) : super(
+          radius,
+          position: position,
+          priority: priority,
+        ) {
+    anchor = Anchor.center;
```

Comment:
Can't `Anchor.center` be made the default for a circle component? Then this class could be removed entirely

Reply:
It would be inconsistent with all the other classes that extend `PositionComponent`.

Reply:
I realized that I haven't implemented `Shape` to take anchor into consideration, so it actually has to be center now for it to work, so I could remove this class.